src/test/compile-fail/borrowck-closures-mut-of-imm.rs

```diff
@@ -24,6 +24,7 @@ fn a(x: &int) {
     //~^ ERROR cannot borrow
     let c2 = || set(&mut *x);
     //~^ ERROR cannot borrow
```

Comment:
Though it might be nice to extend the compile-fail tester with regexp support, so that we could put just the most relevant (and least likely to change) parts in there.